bindings/java/src/org/sleuthkit/datamodel/HostAddressManager.java

```diff
@@ -98,6 +105,12 @@
 	 * @throws TskCoreException
 	 */
 	private Optional<HostAddress> getHostAddress(HostAddress.HostAddressType type, String address, CaseDbConnection connection) throws TskCoreException {
+		
+		HostAddress hostAddress = recentHostAddresstCache.getIfPresent(type.getId() + "#" + address.toLowerCase());
```

Comment:
Can we put the "type.getId() + "#" + address.toLowerCase()" part into a method like "createRecentHostAddressKey()"?